src/Migrator.php

```diff
@@ -228,12 +228,14 @@ private function ensureEmptyTargetTable($table, Connection $sourceConnection, Co
     {
         // SchemaManager doesn't do enums!
         $destConnection->exec('DROP TABLE IF EXISTS ' . $sourceConnection->quoteIdentifier($table));
-
+        $triggerSql = [];
         $driverName = $sourceConnection->getDriver()->getName();
         if ($driverName === 'pdo_mysql') {
             $createSqlRow = $sourceConnection->query('SHOW CREATE TABLE ' . $sourceConnection->quoteIdentifier($table))
                 ->fetch(\PDO::FETCH_ASSOC);
             $createSql = $createSqlRow['Create Table'];
+            // find any triggers
+            $triggerSql = $this->checkTableTriggers($table, $sourceConnection);
```

Comment:
Not sure `check` is such a great verb here, since it's returning / creating the triggers?

Reply:
I have changed to the function name to be more reflective of the functions functionality